bin/s1_read.py

```diff
@@ -1,7 +1,7 @@
 import os
 import sys
 
-from sentinel1_reader import sentinel1_reader, sentinel1_orbit_reader
+from s1reader import reader, orbit
```

Comment:
I'm concerned renaming `sentinel1_reader, sentinel1_orbit_reader` to `reader, orbit` this too vague. @yunjunz What do you think about the following?
```suggestion
from s1reader import s1_reader, s1_orbit
```
 `s1_` is still much shorter than `sentinel1_` and it leaves no ambiguity as to purpose/origin.

Reply:
yeah, that sounds a good idea.

Reply:
I will adjust the naming in the PR accordingly to this.

Reply:
Done.